packages/toast/README.md

```diff
@@ -95,12 +95,12 @@ return (
-| `className`          | `string`                       | Optional className passed to the wrapping `<div />` for the toast.                                                                                                |         |
-| `variant` (Required) | `'success'` \| `'progress'`    | Style variant to render the Toast as.                                                                                                                             |         |
-| `progress`           | `number` (0...1)               | Optional number between 0 and 1 that sets the progress bar's progress. Note that the progress bar is only rendered when the Toast variant is set to `'progress'`. | `1`     |
-| `open`               | `boolean`                      | Optional boolean that renders the Toast and makes it visible when true.                                                                                           | `false` |
-| `close`              | `function` (MouseEventHandler) | Optional click event handler that, when set, renders a close button that receives the passed handler.                                                             |         |
+| Prop                 | Type                                                                    | Description                                                                                                                                                       | Default |
+| -------------------- | ----------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| `title`              | `ReactNode`                                                             | Optional text shown in bold above the body text.                                                                                                                  |         |
+| `body` (Required)    | `ReactNode`                                                             | Main text for the Toast.                                                                                                                                          |         |
+| `className`          | `string`                                                                | Optional className passed to the wrapping `<div />` for the toast.                                                                                                |         |
+| `variant` (Required) | `'success'` \| `'note'` \| `'warning'` \| `'important'` \| `'progress'` | Style variant to render the Toast as.                                                                                                                             |         |
```

Comment:
Moving away from the pipes and in favor of commas to separate variants in readmes to make documentation more human readable